pretty/formatter.test.ts

```diff
@@ -802,3 +803,21 @@ test("getPrettyFormatter() with multiline interpolated values (no align)", () =>
     }
   }
 });
+
+test("renderStructuredValues set to true", () => {
+  const formatter = getPrettyFormatter({
+    renderStructuredValues: true,
+    inspectOptions: { colors: false },
+  });
+
+  const record = createLogRecord("info", ["test"], ["FooBar"], Date.now(), {
+    foo: "bar",
+    bar: "baz",
+  });
+  const result = formatter(record);
+
+  // Should contain multiple lines due to wrapping
+  const lines = result.split("\n");
+  assertEquals(lines.length, 3); // Normal log line + formatted properties + newline
+  assertEquals(lines[1].trim(), '{ foo: "bar", bar: "baz" }');
```

Comment:
I think you need to add a conditional like:

```suggestion
  assertEquals(
    lines[1].trim(),
    "Deno" in globalThis
      ? '{ foo: "bar", bar: "baz" }'
      : "{ foo: 'bar', bar: 'baz' }"
  );
```